build/RepositoryBuild.targets

```diff
@@ -129,7 +129,7 @@
     <Message Text="============ Building $(RepositoryToBuild) ============" Importance="High" />
 
     <Exec
-      Command="./$(_BuildScriptToExecute) -Path $(BuildRepositoryRoot) $(BuildArguments)"
+      Command="./$(_BuildScriptToExecute) -Path $(BuildRepositoryRoot.TrimEnd('\')) $(BuildArguments)"
```

Comment:
Trimming here since the _BuildScriptToExecute expects no trailing slashes but everywhere else in this targets file the BuildRepositoryRoot is used to compute other paths and require a trailing slash. I'm not gonna try to make things more consistent and will just fix the path issues whenever they come up in the easiest way possible.